### ISSUE TITLE
PDF posters about accessibility

### ISSUE THREAD

**Issue by Indu-Kaila** [nhsuk/nhsuk-service-manual#1328]
**What**
Poster PDFs in accessibility and design principles guidance (@karlgoldstraw has offered to do this.) Would be great if we can find HTML versions of these already created as discussed at the meeting – Indu
The GDS/GOV.UK PDF posters about accessibility included within the service manual were not well received in the recent service manual user testing. They were criticised as 
A better way of presenting these PDF posters is needed (maybe via a preview feature). Note: Not a priority ticket at the moment.

**Why**
The GDS/GOV.UK PDF posters about accessibility included within the service manual were not well received and were criticised in the recent service manual user testing as PDFs are not accessible documents.

**Done when**
We have:

- [ ] A way of displaying the PDF posters has been done/created that prevents the problem of highlighting that PDFs are not an accessible document format
- [ ] Could refer to how the GOV.UK site present their PDF accessibility posters

**Comment by karlgoldstraw** [comment]
The PDF accessibility posters are not in an accessible format. I think the solution here might be to create a page dedicated to the posters so that we can explain with text what each poster says. Exactly like they do here: https://accessibility.blog.gov.uk/2016/09/02/dos-and-donts-on-designing-for-accessibility/

Our url would be something like: https://service-manual.nhs.uk/accessibility/tips-for-professions

The page formatting would be something like:

# Accessibility posters

We have 6 PDF posters for printing that contain information on a coloured background explaining basic principles for accessibility in UCD professions. We recommend that you print them and place them in a visible space in your place of work to help others find out what they can do and where to go to find out more at: www.service-manual.nhs.uk/accessibility

[screenshot of all posters as a jpg + caption]

## What the posters say

### Product and delivery

Make sure everyone knows their responsibilities. If a team member does not have a basic understanding already, help them get started with accessibility. (Includes an image of two hands holding a phone on a blue background).

### User research

Involve people with access needs at every stage. Understand participants’ needs beforehand and make your sessions accessible on the day. (Include an image of research notes on a yellow background).

### Content

Write content that’s inclusive and easy to understand. Use alternative text for images in content – and use captions to complement alt-text. (Includes an image of a phone with text on it on a pink background).

### Development

Check keyboard accessibility – start with the correct HTML for links, buttons and form controls. Use ARIA patterns to let people who use assistive tech know what’s going on. (Includes an image of a HTML tag on a purple background).

### Design 

Use headings correctly, define focus styles, and build with tested, accessible components. Check colour contrast meets at least level AA and aim for AAA where possible. (Includes an image of two hands making a vista shape on an orange background).

### Testing

Learn about accessibility before you start. Do 3 kinds of tests: test with people with access needs, do automated tests, and manual testing with guidelines. (Includes an image of clipboard with a tick on a green background).

**Comment by sarawilcox** [comment]
Agree with your suggestion, @karlgoldstraw. Just that the URL needs to reflect the page title.

**Comment by ambchang** [comment]
Prototyped Karl's suggestion and is now on review environment waiting @sarawilcox's content review next week.

Link: https://nhsuk-service-manual-pr.azurewebsites.net/accessibility/accessibility-posters

**Comment by sarawilcox** [comment]
Ready to release.

**Comment by sarawilcox** [comment]
Now live. I'll close this ticket.
